js/script.js

```diff
@@ -16,22 +19,27 @@ import {
   showSpinnerOverlay,
   hideSpinnerOverlay,
   formatDate,
+  setActiveButton,
 } from "./utilities.js";
 import { MESSAGES } from "./const.js";
 
 let tasks = [];
 let isVisible = false;
+let currentFilter = "all";
+let filteredOrSearchableTasks = [];
+let tasksDisplayed = 0;
+const tasksPerPage = 9;
```

Comment:
Move these constants to another `const.js` file and then export it

Reply:
`tasksPerPage` constant moved to const.js but rest of the things needs to be in this file.